### DISCUSSION TITLE
Issues with Welcome and Goodbye event execution.

### DISCUSSION THREAD

**Discussion by muhitrhn** [Androz2091/AtlantaBot#471] [Q&A] [answered]
Hi, hope everyone is healthy in these pandemic times.

So, i need help with this. For some reason, `welcome` and `goodbye` commands doesn't trigger the `guildMemberAdd` and `guildMemberRemove` events for me. I have ran `welcome test` a few times but i don't see any error in log. As like everything is working as they should. When running  the `welcome test`, i only see Test Executed message but no test preview on welcome channel.

Can anyone help me fix this issue? Thank you in advance!

N.B. - I have both `PRESENCE INTENT` and `GUILD MEMBER INTENTS` enabled on my bot.

**Comment by Androz2091** [comment, marked as answer]
This has been fixed with the v4.11.0 released :wink:

**Reply by ShrlGnwn** [reply]
welcome message not working in v4.12.x?
